app/core/config/routes.rb

```diff
@@ -7,5 +7,9 @@
 
   resources :results
 
+  resources :leagues, only: [] do
```

Comment:
the only is not needed.. I have just only open routes that I have implemented.. make it easy to keep track of what routes exist in a single file.. once we have league routes it will be far nicer..